### PR TITLE
Build bundles with rollup

### PR DESCRIPTION
**Pull request by TrySound**
Similar to jss setup.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,14 +21,15 @@
     "css-in-js"
   ],
   "engines": {},
+  "main": "./dist/css-vendor.cjs.js",
+  "module": "./dist/css-vendor.esm.js",
+  "files": [
+    "dist"
```
Comment (kof) [inline package.json]: we also publish src and LICENSE in JSS core https://github.com/cssinjs/jss/blob/master/packages/jss/package.json#L22

File: package.json [hunk 2]
```diff
@@ -25,7 +25,8 @@
   "module": "./dist/css-vendor.esm.js",
   "files": [
     "dist",
-    "src"
+    "src",
+    "LICENSE"
```
Comment (HenriBeck) [inline package.json]: License will always be published: https://docs.npmjs.com/files/package.json.html#files

Comment (kof) [reply package.json]: cool, didn't know

File: package.json (hunk 1 above)
Comment (HenriBeck) [reply package.json]: Src is only published for flow types, but this repo doesn't have them, so there is no need to publish the source code.

**Comment by TrySound** [comment]
I think it's ready

**Pull request merged**